client/realm_management_test.go

```diff
@@ -113,6 +114,41 @@ func TestInstallInterface(t *testing.T) {
 	}
 }
 
+func TestInstallInterface_URL(t *testing.T) {
+	testIface, _ := interfaces.ParseInterface([]byte(testInterface))
+	c, _ := getTestContext(t)
+
+	tests := []struct {
+		name    string
+		isAsync bool
+		url     string
+	}{
+		{
+			name:    "Async operation",
+			isAsync: true,
+			url:     "/realmmanagement/v1/test/interfaces'",
```

Comment:
The reason why I am doing this is due to the response of `.ToCurl(c)`:

```
...-H 'User-Agent: astarte-go' 'http://127.0.0.1:36749/realmmanagement/v1/test/interfaces'
```

If we use `/realmmanagement/v1/test/interfaces` (without `'`), it will pass for both cases of:
- `/realmmanagement/v1/test/interfaces`
- `/realmmanagement/v1/test/interfaces?async_operation=true`

Because both of them contain `/realmmanagement/v1/test/interfaces`

Im using the `'` to make sure that we don't have an url of `/realmmanagement/v1/test/interfaces?async_operation=true`

Wdyt?

Reply:
```
tests := []struct {
	name    string
	isAsync bool
	url     string
}{
	{
		name:    "Async operation",
		isAsync: true,
		url:     "/realmmanagement/v1/test/interfaces'",
	},
```
In addition if we remove the `'`, the test above will also pass for `/realmmanagement/v1/test/interfaces?async_operation=true` (where we expect that it should not pass)

Reply:
Ok, fair enough. You can leave it as it is. :+1: